xml/obs_authorization_token.xml

```diff
@@ -84,6 +84,16 @@
    <screen><command>osc</command> token --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
    <screen><command>osc</command> token --trigger &lt;TOKEN&gt;</screen>
    <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/runservice</screen>
+
+   <para>A token to rebuild a package can be created and trigger by </para>
+   <screen><command>osc</command> token --opertion rebuild --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
+   <screen><command>osc</command> token --operation rebuild --trigger &lt;TOKEN&gt;</screen>
+   <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/rebuild</screen>
```

Comment:
There is no need to have separate screens. Better combine all the lines into one. Also use `replaceable`:

```suggestion
   <screen><command>osc</command> token --opertion rebuild --create <replaceable>PROJECT</replaceable> <replaceable>PACKAGE</replaceable>
<command>osc</command> token --operation rebuild --trigger <replaceable>TOKEN</replaceable>
<command>curl</command> -H "Authorization: Token <replaceable>TOKEN</replaceable>" -X POST /trigger/rebuild</screen>
```